desktop/apps/partner2/stylesheets/overview.styl

```diff
@@ -7,6 +7,7 @@
     margin-bottom 60px
     border-top 1px solid gray-color
     flex 0 1 100%
+    width 100%
```

Comment:
From my limited knowledge, we should be able to just use `flex-basis` alone. But in this case, it seems we still need `width` to help calculate the overflowing children's %-based width. Also, flexbox has many cross-browser [issues](https://github.com/philipwalton/flexbugs), so 🤷‍♂️